sdk/lib/opentelemetry/sdk/configurator.rb

```diff
@@ -170,6 +172,7 @@ def configure_span_processors
       end
 
       def wrapped_exporters_from_env # rubocop:disable Metrics/CyclomaticComplexity, Metrics/AbcSize
+        metrics_reporter = new_metrics_reporter
```

Comment:
We're calling `new_metrics_reporter` in `configure` as well, but I don't see us using the `@metrics_reporter` - is that intentional?

Reply:
Done